diff --git a/dist/timestamp.js b/dist/timestamp.js
index e83c85f3265dc5a10023e308f218700d9f142d9f..196a1e87851208eb8255456da0b12485d55f1cbf 100644
--- a/dist/timestamp.js
+++ b/dist/timestamp.js
@@ -1,4 +1,3 @@
-import { zonedTimeToUtc } from 'date-fns-tz';
 import { read } from 'text-kit';
 export const parse = (input, { timezone = Intl.DateTimeFormat().resolvedOptions().timeZone } = {}) => {
     const { match, eat, getChar, jump } = read(input);
@@ -32,8 +31,8 @@ export const parse = (input, { timezone = Intl.DateTimeFormat().resolvedOptions(
             (opening.value === '<' && closing === '>')) {
             eat('char');
             return {
-                date: zonedTimeToUtc(date, timezone),
-                end: end ? zonedTimeToUtc(end, timezone) : undefined
+                date: new Date(Number.NaN),
+                end: end ? new Date(Number.NaN) : undefined
             };
         }
         // opening closing does not match
